src/heuristics/cargo_target.rs

```diff
@@ -12,7 +12,7 @@ impl Heuristic for CargoTarget {
 
     fn check_for_matches(&self, state: &mut MatchingState) {
         if state.has_file("Cargo.toml").is_some() && state.has_directory("target").is_some() {
-            state.add_weight_with_comment("target", 1000, "Found Cargo.toml and target directory.");
+            state.add_match("target", "Found Cargo.toml and target directory.");
```

Comment:
we have builder and don't use it?

Reply:
Yes, because its defaults are good enough for this heuristic. You should chain more builder method calls only if you need it to assign a custom weight for example.